### PR TITLE
Style orgs list in user profile

### PR DESCRIPTION
**Pull request by xf-**
**Change:**
Add a wrapper to organization list to target it with css selector, remove fixed size. Add CSS for organization list.
I use important because some really heavy large selector setting some options.

**Current state:**
The organization list has whitespaces instead of padding and no space to second row

### CODE REVIEW

**Comment by xf-** [comment]
Added -3px margin to remove the padding on borders or the list.

**Comment by lafriks** [comment]
Screenshot old vs new?

**Review by kolaente — COMMENTED** [review]
Looks good, could you add some screenshots? (From the whole org page would be great)

**Comment by xf-** [comment]
@lafriks my screenshots are gone in PR Oo i send some in discord
@kolaente yes. any specific breakpoints, sizes?

File: public/less/_user.less [hunk 1]
```diff
@@ -63,6 +63,21 @@
         }
     }
 
+    &-orgs {
```
Comment (lafriks) [inline public/less/_user.less]: I would prefer that full class name was here `.user-orgs`

Comment (xf-) [reply public/less/_user.less]: OK i will change it, but i prefer smaller css selectors. Easier to work with and i see not huge differences to selector like:
```scss
.user {
&.profile {
}
}
```
This is much easier to overwrite/adjust compared to `.user.profile .ui.card .extra.content ul li:not(:last-child)`.

Comment (lafriks) [reply public/less/_user.less]: Me too but I don't quite like that single class can not be found in less files this way

Comment (xf-) [reply public/less/_user.less]: @lafriks 
i will change it after https://github.com/go-gitea/gitea/pull/6919 is merged or closed. makes much easier to create PRs

**Review by lafriks — APPROVED** [review]

**Review by kolaente — APPROVED** [review]

**Pull request merged**